jbpm-audit/pom.xml

```diff
@@ -67,6 +67,10 @@
       <groupId>com.thoughtworks.xstream</groupId>
       <artifactId>xstream</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.narayana.jta</groupId>
+      <artifactId>narayana-jta</artifactId>
```

Comment:
shouldn't this be in test scope? same as the removed btw was? Otherwise it will be included in all other assemblies like for EAP or WAS/Weblogic